pyspinw/gui/coordinateeditor.py

```diff
@@ -0,0 +1,267 @@
+import numpy as np
+from PySide6.QtCore import Signal
+from PySide6.QtWidgets import QWidget, QVBoxLayout, QGridLayout, QApplication, QLabel

```

Comment:
Could we use the [qtpy](https://pypi.org/project/QtPy/) shim instead of PySide/PyQt directly please? The syntax is almost identical but would allow users to use PyQt6 or PySide6 (or even PyQt5/PySide2) as they prefer (or already have installed on their system).

Reply:
This sounds doable, will need to check a couple of things regarding Qt5 and Qt6 before a definite yes.